From 7f3cc9a623b33e61478055ede2b5e5bab1d70502 Mon Sep 17 00:00:00 2001
From: AdityaOza999 <aditya.oza@teksun.com>
Date: Thu, 5 Nov 2020 14:50:45 +0530
Subject: [PATCH] 3

---
 test1.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test1.txt b/test1.txt
index 9d37acf..316d92a 100644
--- a/test1.txt
+++ b/test1.txt
@@ -2,3 +2,4 @@
 
 First master line
 second master line
+third master line************
-- 
2.25.1

